prov/sockets/src/sock_progress.c

```diff
@@ -2464,6 +2463,11 @@ static void sock_thread_set_affinity(char *s)
 	int j, first, last,stride;
 	cpu_set_t mycpuset;
 	pthread_t mythread;
+ 	
+	if(!s) {
+		SOCK_LOG_DBG("Invalid FI_SOCKETS_PE_AFFINITY value\n");
+		return;
+	}
```

Comment:
I also found that the check is not needed but since it was reported by coverity, I added it. Thanks for reporting it in coverity.